test/integration/reg_user_can_view_past_order_test.rb

```diff
@@ -0,0 +1,73 @@
+require 'test_helper'
+
+
+class RegUserCanViewPastOrderTest < ActionDispatch::IntegrationTest
+
+  # Background: An existing user that has one previous order
+
+  test 'a registered user can view past orders' do
+
+    user = create_user
+    create_category_and_items(1)
+    item1 = Item.find_by(title: "pour over0")
+    item2 = Item.find_by(title: "drip0")
+
+    visit items_path
+    within "#item#{item1.id}" do
+      click_button "Add to Cart"
+    end
+
+    within "#item#{item2.id}" do
+      click_button "Add to Cart"
+    end
+
+    click_button "View Cart"
+    click_button "Checkout"
+    fill_in 'Username', with: 'ryan'
+    fill_in "Password", with: 'waffles'
+    click_button "Login"
+
+    assert '/dashboard', current_path
+
+    click_link "Cart"
+    click_button "Checkout"
+
+    order1 = user.orders.last
+
+    click_link "Logout"
+    click_link "Login"
+
+    fill_in 'Username', with: 'ryan'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.